src/components/List.vue

```diff
@@ -11,6 +11,10 @@ export default {
   name: 'list',
   props: [
     'items'
-  ]
+  ],
+  mounted() {
+    console.error('throwing an error');
+    throw('Error');
+  }
```

Comment:
I'm not sure if the example makes sense - you'd expect your component to function properly and never throw errors, no?

Reply:
Correct, this example is boiling down the ultimate issue whereas the real component that led to the issue is more complex and can go into detail on that if you need more info on it.

At least in my view, I would want that error to at bubble up over being swallowed and warned to the console.  It would prevent a critical error from giving a false green test suite.